core/domain/question_services.py

```diff
@@ -153,6 +153,63 @@ def get_question_by_id(question_id, strict=True):
         return None
 
 
+def get_question_skill_links_of_skill(skill_id):
+    """Returns a list of QuestionSkillLink domains of a particular skill ID.
+
+    Args:
+        skill_id: str. ID of the skill
+
+    Returns:
+        list(QuestionSkillLink)|None. The list of question skill link
+        domains that are linked to the skill ID. None if it doesnt exist.
```

Comment:
domains --> domain objects

"skill ID. None if it" --> "skill ID, or None if the skill ID does not exist." (rationale: doesnt is misspelled; clarify what "it" refers to)